src/common/execution.py

```diff
@@ -149,8 +149,46 @@ async def get_oracles() -> Oracles:
     )
 
 
-async def check_gas_price() -> bool:
-    max_fee_per_gas = await _get_max_fee_per_gas()
+async def get_tx_params() -> TxParams:
```

Comment:
this tx params relevant only to validator's registration transactions. MB move to validators/execution.py and rename to smth like get_validator_tx_params?

Reply:
Or this actual to high priority transaction? Also can be expressed in the func name

Reply:
get_tx_params is used by check_gas_price located in common/execution

If I move get_tx_params to validators dir then common/execution file will import validators/execution. The imports were done in another order so far.

Reply:
get_tx_params name is generic, so yes, high priority should be expressed